src/js/media-container.js

```diff
@@ -209,6 +209,14 @@ function getBreakpoints(breakpoints, rect) {
 /**
  * @extends {HTMLElement}
  *
+ * @attr {boolean} audio
+ * @attr {string} autohide
+ * @attr {string} breakpoints
+ * @attr {boolean} gesturesdisabled
+ * @attr {boolean} keyboardcontrol
+ * @attr {boolean} noautohide
+ * @attr {boolean} userinactive
```

Comment:
I figured media-controller and media-container we'll probably do manually to add more explanation